src/map/entities/automatonentity.cpp

```diff
@@ -19,26 +19,24 @@
 ===========================================================================
 */
 
-#include "automatonentity.h"
+#include <memory>
+
 #include "../ai/ai_container.h"
-#include "../ai/controllers/automaton_controller.h"
```

Comment:
Does this not still need the automaton controller?  utils.h?

Reply:
My ide pointed out that it didn't, so I removed it. (I did test build / run everything as usual)